src/styles/_catalog.scss

```diff
@@ -15,8 +28,4 @@
```

Comment:
Since the 'justify-content: center' property is not necessary for the card layout, consider removing it to simplify the code.